clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -1210,7 +1210,7 @@ public void handleSuccessfulResponse(RequestHeader requestHeader, long now, Meta
             // Check if any topic's metadata failed to get updated
             Map<String, Errors> errors = response.errors();
             if (!errors.isEmpty())
-                log.warn("Error while fetching metadata with correlation id {} : {}", requestHeader.correlationId(), errors);
+                log.warn("The metadata response from the cluster reported a recoverable issue with correlation id {} : {}", requestHeader.correlationId(), response.errors());
```

Comment:
Please reuse `errors` to avoid creating collection again.

Reply:
@chia7712 - Mentioned changes has been done. Kindly review it.